common/djangoapps/student/tests/test_models.py

```diff
@@ -403,3 +403,112 @@ def test_retire_recovery_email(self):
 
         # Assert that there is no longer an AccountRecovery record for this user
         assert len(AccountRecovery.objects.filter(user_id=user.id)) == 0
+
+@ddt.ddt
+class TestUserPostSaveCallback(SharedModuleStoreTestCase):
+    def setUp(self):
```

Comment:
We could probably move this into the helper function but then we'd need to deal with a tuple of two objects on the return, so I'm not sure it's worth moving it (this will get called once per test in either case). Happy to move if anyone has strong preferences, or refactor into something more sensible (e.g. a global-to-the-test object that persists between individual unit tests)